garage.toml

```diff
@@ -0,0 +1,29 @@
+metadata_dir = "/var/lib/garage/meta"
+data_dir = "/var/lib/garage/data"
+db_engine = "sqlite"
+
+replication_factor = 1
+# idk why rcp_ is needed without replication but ok
+rpc_bind_addr = "[::]:3901"
+rpc_public_addr = "127.0.0.1:3901"
```

Comment:
Since this is on Docker, I think it's better if we use `0.0.0.0:3901` as the hostname, or just `:3901`

Reply:
This entire `rpc_` section is obsolete as we are in single-node mode. Otherwise I think you are right.